go/integrityblock/hash.go

```diff
@@ -0,0 +1,22 @@
+package integrityblock
```

Comment:
Now this file contains only one pretty simple function. Can we merge this into integrityblock.go?